### DISCUSSION TITLE
call action modal close

### DISCUSSION THREAD

**Discussion by asharifauzan** [saadeghi/daisyui#956] [Q&A] [answered]
Hello, I want to create a modal with a form inside. So when the button submit is clicked it will close the modal. I'm using Reactjs and doing this job I should create a react state manually to handle modal visibility. Is tailwind provide a function like bootstrap  `myModal.modal('hide)` ? Thankyou Anyway

**Comment by asharifauzan** [comment, marked as answer]
UPDATE
I'm using useRef, so when the submit button is fired it will executes `closeModalBtnRef.current.click()`

**Reply by rhscjohn-dev** [reply]
Check out the Daisyui  Modal Class Name `model-open`.  Add/remove this class to open/close the modal using Javascript.  See [Modal docs](https://daisyui.com/components/modal/).

**Reply by yaffalhakim1** [reply]
do you have an example code using useRef? i faced the same problem
